gst/nnstreamer/tensor_filter/tensor_filter_common.c

```diff
@@ -2425,6 +2427,13 @@ gst_tensor_filter_common_open_fw (GstTensorFilterPrivate * priv)
     } else {
       priv->prop.fw_opened = TRUE;
     }
+
+    stop_time = g_get_monotonic_time ();
+    if (priv->prop.fwname && priv->prop.model_files) {
```

Comment:
No need to check `priv->prop.fw_opened == TRUE`?

Reply:
Right, No need to log when the opening fails. I added the condition, Thanks!